react/components/ShippingOptionList.tsx

```diff
@@ -66,7 +76,12 @@ const ShippingOptionList: React.FC<Props> = ({
             <div className="flex w-100">
               <div className="flex flex-column w-100">
                 <span className="c-on-base">{deliveryOption.id}</span>
-                <span className="c-muted-1">
+                <span
+                  className={classnames({
+                    'c-muted-1': deliveryOption.id !== fastestOption?.id,
+                    'c-success': deliveryOption.id === fastestOption?.id,
```

Comment:
I think this is more of an UX issue, but how can the user know why this is green? Maybe a tooltip would do it?